spk/syncthing/src/ca_reloader.sh

```diff
@@ -10,16 +10,16 @@ SYNCTHING=/var/packages/syncthing/target/bin/syncthing
 case $1 in
     syncthing_webui)
         # Forcefully overwrite certificate files by symlinks
-        ln -sf $CERT_DIR/cert.pem $CONF_DIR/https-cert.pem
-        ln -sf $CERT_DIR/privkey.pem $CONF_DIR/https-key.pem
+        ln -sf ${CERT_DIR}/cert.pem ${CONF_DIR}/https-cert.pem
+        ln -sf ${CERT_DIR}/privkey.pem ${CONF_DIR}/https-key.pem
 
         # Required: set $HOME environment variable
-        HOME=$CONF_DIR
+        HOME=${CONF_DIR}
         export HOME
 
-        $SYNCTHING cli --home=$CONF_DIR operations restart
+        ${SYNCTHING} cli --config=${CONF_DIR} --data=${CONF_DIR} operations restart
```

Comment:
This doesn't really change anything, the options are equivalent.  Is it just to avoid confusion between `$HOME` and `--home`?

Reply:
Yes it is (but it is not equivalent for `syncthing generate` that does not accept the --data parameter)
I was very confused until I realized that `--home` does not define the `HOME` folder.